app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/service.tsx

```diff
@@ -72,6 +159,77 @@ export class Backend {
         }
         return response.model<Organization>();
       });
+
+  /** Returns a list of projects belonging to the current user, from the Cloud backend API. */
+  listProjects = (): Promise<Project[]> =>
+    this.get(LIST_PROJECTS_PATH)
+      .send()
+      .then(async (response) => {
+        if (response.status() === 401 || response.status() === 404) {
+          return [];
```

Comment:
is there a reason this one doesn't throw an error?

Reply:
Not that I can recall. Must be a missed refactor. Good catch.